Jucer2Reprojucer/main.cpp

```diff
@@ -1113,10 +1113,10 @@ int main(int argc, char* argv[])
       const auto isVstPluginHost = jucerProject.getChildWithName("MODULES")
                                      .getChildWithProperty("id", "juce_audio_processors")
                                      .isValid()
-                                   && jucerProject.getChildWithName("JUCEOPTIONS")
+                                   && juce::StringArray{"enabled", "1"}.contains(
```

Comment:
This one is quite tricky to read, why would a compile time StringArray contain run time variable ?

That's because the intent is : this runtime variable have one of the values from this compile time StringArray

Reply:
@MartyLake would you prefer the following?
```diff
diff --git a/Jucer2Reprojucer/main.cpp b/Jucer2Reprojucer/main.cpp
index f77a4844..bef7d7bc 100644
--- a/Jucer2Reprojucer/main.cpp
+++ b/Jucer2Reprojucer/main.cpp
@@ -1122,13 +1122,14 @@ int main(int argc, char* argv[])
       const auto hasVst2Interface = jucerVersionAsTuple > Version{4, 2, 3};
       const auto isVstAudioPlugin =
         isAudioPlugin && bool{jucerProject.getProperty("buildVST")};
-      const auto isVstPluginHost = jucerProject.getChildWithName("MODULES")
-                                     .getChildWithProperty("id", "juce_audio_processors")
-                                     .isValid()
-                                   && juce::StringArray{"enabled", "1"}.contains(
-                                        jucerProject.getChildWithName("JUCEOPTIONS")
-                                          .getProperty("JUCE_PLUGINHOST_VST")
-                                          .toString());
+      const auto pluginHostVstOption = jucerProject.getChildWithName("JUCEOPTIONS")
+                                         .getProperty("JUCE_PLUGINHOST_VST")
+                                         .toString();
+      const auto isVstPluginHost =
+        jucerProject.getChildWithName("MODULES")
+          .getChildWithProperty("id", "juce_audio_processors")
+          .isValid()
+        && (pluginHostVstOption == "enabled" || pluginHostVstOption == "1");
 
       if (!hasVst2Interface && (isVstAudioPlugin || isVstPluginHost))
       {
```